penpot/penpot
# :tada: Access Tokens (part 1) and bugfixes

This PR has:
- initial approach for access tokens authentication (part 1)
- move media mutations to commands (mutations->commands migration)
- fix some internal inconsistencies on comments rpc methods
- fix not working database error reporter (regression in this release)
- fix not working ldap authentication (regression in this release)

How to test:
1. the comments part just test if you can comment, create comment, delete comment, update comments. This part was changed a lot because previously there are no proper permissions check on all methods and many other issues.
2. the media mutations, just upload a file, nothing changed just moved
3. database error reporter: right now all reported errors through the mattermost integration does not work (staging), because they are not persisting on db. with pr fixes this
4. the ldap was broken, it should work (I have tested it locally so, you don't need to test it)

The access tokens part still has missing stuff, there are no tests and no rpc methods, this us just work in progress, no testing needed.
